src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -442,6 +442,14 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
                             result = GetResultForEnumPropertyValueOfDSCResource(completionContext, string.Empty, ref replacementIndex, ref replacementLength, out unused);
                         }
                         break;
+                    case TokenKind.AtCurly:
+                        // Handle scenarios such as Sort-Object @{<tab> || gci | Format-Table @{
```

Comment:
> Sort-Object @{<tab> || gci | Format-Table @{

minor comment: can we change this to `'Sort-Object @{<tab>' or 'gci | Format-Table @{'`? I think it's more readable.